client/src/assets/images/menu/language.svg

```diff
@@ -0,0 +1,10 @@
+<!-- by Aaron Jin - free for commercial use -->
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200" style="transform:scale(1.2)">
```

Comment:
Please add Aaron Jin in the Credits. What is the licence of this icon?

Reply:
the license is "free for commercial use", as stated…
see https://icon-icons.com/icon/Language-transtlation-translate/111234